### PR TITLE
[demo] add a distributed training demo on movielens for dynamic_embedding

### PR DESCRIPTION
**Pull request by rhdong**
[demo] add a distributed training demo on movielens for dynamic_embedding

### CODE REVIEW

File: demo/movielens-1m-ps/movielens-1m-ps.py [hunk 1]
```diff
@@ -0,0 +1,214 @@
+import os, sys
+import tensorflow as tf
+from tensorflow.keras.layers import Dense
+import tensorflow_datasets as tfds
+import tensorflow_recommenders_addons as tfra
+
+tf.compat.v1.disable_v2_behavior()
```
Comment (Lifann) [inline demo/movielens-1m-ps/movielens-1m-ps.py:7]: `disable_eager_execution` and `disable_resource_variables ` are already called in `disable_v2_behavior `. [here](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/compat/v2_compat.py#L94-L96)

File: demo/movielens-1m-ps/movielens-1m-ps.py [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+  task_id = FLAGS.task_id
+
+  print('ps_list: ', ps_list)
+  print('worker_list: ', worker_list)
+
+  if task_mode == 'ps':
+    ps_config = {"cluster": {"ps": ps_list, "worker": worker_list}}
+    start_ps(task_id, ps_config)
+  elif task_mode == 'worker':
+    worker_config = {"cluster": {"ps": ps_list, "worker": worker_list}}
```
Comment (Lifann) [inline demo/movielens-1m-ps/movielens-1m-ps.py:206]: `ps_config` and `worker_config` seem to be the same. Use only `cluster_config` instead.

File: demo/movielens-1m-ps/movielens-1m-ps.py (hunk 1 above)
Comment (rhdong) [reply demo/movielens-1m-ps/movielens-1m-ps.py:7]: accept

File: demo/movielens-1m-ps/movielens-1m-ps.py (hunk 2 above)
Comment (rhdong) [reply demo/movielens-1m-ps/movielens-1m-ps.py:206]: accept

**Review by Lifann — APPROVED** [review]
LGTM

**Pull request merged**